typed-racket-lib/typed-racket/base-env/base-special-env.rkt

```diff
@@ -5,6 +5,7 @@
 (require "../env/init-envs.rkt"
          "../types/abbrev.rkt"
          "../types/numeric-tower.rkt"
+	 (for-template (submod racket/file proc-names))
```

Comment:
Would using `make-template-identifier` (like other identifiers) be better? I worry about using `for-template`, because it can easily create a lot more module dependencies and instantiations than intended.

Reply:
The reason I'm not using `make-template-identifier` is that the identifier I want has the same name as the `define-syntax` binding so I don't think I can reliably pick out the right one.

Reply:
Latest approach makes sense!